api/src/shared/infrastructure/repositories/answer-repository.js

```diff
@@ -81,6 +81,16 @@ const findByAssessment = async function (assessmentId) {
   return _toDomainArray(answerDTOsWithoutDuplicate);
 };
 
+const findByAssessmentExcludingChallengeIds = async function ({ assessmentId, excludedChallengeIds = [] }) {
+  const answerDTOs = await knex
```

Comment:
J'aurais bien mis sous format getConnection() tout de meme. Mais ca obligerait a refaire le test func ^^

Reply:
Noté pour une prochaine fois :)